addons/account/models/account_move.py

```diff
@@ -2601,6 +2606,9 @@ def _onchange_quick_edit_total_amount(self):
 
     @api.onchange('invoice_line_ids')
     def _onchange_quick_edit_line_ids(self):
+        with self._sync_rounding_lines({'records': self}):
```

Comment:
seems a little odd to call the @contextmanager method like that, if it's necessary maybe it would be more clear to call  `_recompute_cash_rounding_lines` directly (or just add invoice_line_ids in dependencies of _recompute_cash_rounding_lines

Reply:
Yeah, that's weird and it doesn't even work correctly. 😆 

Actually, I talked to William (wan) this, and it seems this issue can't be fixed. 
But I completely forgot to close this PR.  

Thanks for the reminder.